pkgs/development/python-modules/pycrdt-websocket/default.nix

```diff
@@ -26,18 +26,22 @@
 
 buildPythonPackage rec {
   pname = "pycrdt-websocket";
-  version = "0.15.1";
+  version = "0.15.3";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "jupyter-server";
     repo = "pycrdt-websocket";
     tag = "v${version}";
-    hash = "sha256-O0GRk81at8bgv+/4au8A55dZK2A28+ghy3sitAAZQBI=";
+    hash = "sha256-VK4nSsxF/FbCnyS1RM/JHCPS0Omj4rvdwjzLNDq2hjk=";
   };
 
   build-system = [ hatchling ];
 
+  pythonRelaxDeps = [
+    "pycrdt"
+  ];
```

Comment:
Unless other packages require a newer version of pycrdt, I don't see the need to upgrade by relaxing the restrictions.
It may be a meaningless upper bound, but these two libraries are tied together and I would like to follow them upstream as much as possible.